### DISCUSSION TITLE
Build TTGO_T-listener fails due to hal/gpio_ll.h

### DISCUSSION THREAD

**Discussion by Jocke4f** [josephdadams/TallyArbiter#614] [Q&A] [answered]
Building TTGO_T-listeners have failed for some time. It looks like it is caused by the change in TFT_eSPI in v2.5.34. This version handles a change done by the Espressif software team. As a consequence is the following added
`#include "hal/gpio_ll.h"`
This include breaks the build of the TTGO_T-listeners.

By changing in build-listener-clients.yaml
`git clone https://github.com/Bodmer/TFT_eSPI/`
to
`git clone --branch v2.5.33 https://github.com/Bodmer/TFT_eSPI/`
can the TTGO_T-listeners be built. I don't have the hardware so I can't test if they work or not.

I didn't manage pin point the root cause to why it failed when version 2.5.34 was used.

Would it be a temporary workaround by forcie the build to use v2.5.33 or is the only viable solution to adapt the code so that it works with v2.5.34?

**Comment by josephdadams** [comment]
Fine with me to force the version

**Reply by Jocke4f** [reply, marked as answer]
Ok. I have created PR https://github.com/josephdadams/TallyArbiter/pull/617 for this.
